### ISSUE TITLE
Not compatible with gnu-tar 1.30

### ISSUE THREAD

**Issue by lucasm-iRonin** [heroku/heroku-builds#64]
This plugins was working with `gnu-tar 1.29_1` on OS X but after upgrade to `1.30` it does not work anymore:

```
gtar: --exclude-vcs-ignores has no effect
gtar: Exiting with failure status due to previous errors
 ▸    Command failed: /usr/local/bin/gtar cz -C /Users/lucas/Code/talent-management-system/web/dist --exclude .git --exclude .gitmodules .
 ▸    --exclude-vcs-ignores > /var/folders/r1/c_lfj_yd1bq94qwvfdr6rq0m0000gn/T/f4a64bdb-317e-4427-b7c7-05ca56221790.tar.gz
 ▸    gtar: The following options were used after any non-optional arguments in archive create or update mode.  These options are positional
 ▸    and affect only arguments that follow them.  Please, rearrange them properly.
 ▸    gtar: --exclude-vcs-ignores has no effect
 ▸    gtar: Exiting with failure status due to previous errors
 ▸
```

**Comment by dmathieu** [comment]
Thank you for reporting this issue Lucas. It should be fixed in #65.

**Comment by lucasm-iRonin** [comment]
@dmathieu thanks for quick reaction. I will check it after #65 get merged.

**Comment by dmathieu** [comment]
Fixed in #65.
